services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -235,7 +241,27 @@ export class WorkflowService {
     workflow: TWorkflowWithRelations,
     addNextEvents = true,
   ): TWorkflowWithRelations {
-    const isIndividual = 'endUser' in workflow;
+    const getEntity = (workflow: TWorkflowWithRelations) => {
```

Comment:
Previously we only checked if the `endUser` property exists, this broke when `endUser` was `null`. Also in other places we should not assume that if an entity is not an individual its a business and throw an error instead.